app/components/sidebar/SideBar.tsx

```diff
@@ -16,10 +16,11 @@ const ThemedSidebar = styled(Sidebar)`
         background-color: ${props => props.theme.sidebarColor} !important;
         display: flex;
         overflow-y: visible !important;
-        ${props => (!props.expanded && `width: ${collapsedSidebarWidth} !important;`) || ''}
+        ${props => (!props.$expanded && `width: ${collapsedSidebarWidth} !important;`) || ''}
```

Comment:
Optional:

Just wondering why not:
```suggestion
        ${props => (!props.$expanded ? `width: ${collapsedSidebarWidth} !important;` : ''}
```
?